tests/__init__.py

```diff
@@ -884,11 +891,21 @@ def wrapper(wrapped, instance, args, kwargs):
         except Exception as e:
             # Even though it's unlikely any traces have been sent, make the
             # final request to the test agent so that the test case is finished.
-            conn = httplib.HTTPConnection(tracer.writer.api.hostname, tracer.writer.api.port)
+            conn = httplib.HTTPConnection(tracer.writer._hostname, tracer.writer._port)
             conn.request("GET", "/test/snapshot?ignores=%s&token=%s" % (",".join(ignores), token))
             conn.getresponse()
             pytest.fail("Unexpected test failure during snapshot test: %s" % str(e), pytrace=True)
         finally:
             conn.close()
 
     return wrapper
+
+
+class AnyStr(object):
+    def __eq__(self, other):
+        return isinstance(other, str)
+
+
+class AnyInt(object):
+    def __eq__(self, other):
+        return isinstance(other, int)
```

Comment:
?

Reply:
See the tests for why these exist. Happy to consider alternative solutions though